draft-mdt-quic-explicit-measurements.md

```diff
@@ -379,6 +398,12 @@ method.  Latency spin bit signal edge can be used for the same purpose.
 
 # Security Considerations
 
+The measurements described in this document do not imply new packets injected
+into the network causing potential harm to the network itself and to data
+traffic. The measurements could be harmed by an attacker altering the marking
+of the packets or injecting artificial traffic. Authentication techniques can be
+used where appropriate to guard against these traffic attacks.
```

Comment:
> Authentication techniques can be used where appropriate to guard against these traffic attacks.

This is likely to raise eyebrows due to imprecision. This is a guide to implementers -- they will want to know the details of at least one technique they can use.